Microsoft.Toolkit.Uwp.UI/Extensions/TextBoxMask/TextBoxMask.Properties.cs

```diff
@@ -35,6 +35,9 @@ public partial class TextBoxMask
         private static readonly DependencyProperty OldSelectionLengthProperty = DependencyProperty.RegisterAttached("OldSelectionLength", typeof(int), typeof(TextBoxMask), new PropertyMetadata(0));
         private static readonly DependencyProperty OldSelectionStartProperty = DependencyProperty.RegisterAttached("OldSelectionStart", typeof(int), typeof(TextBoxMask), new PropertyMetadata(0));
 
+        private static readonly DependencyProperty EscapedMaskProperty = DependencyProperty.RegisterAttached("EscapedMask", typeof(string), typeof(TextBoxMask), new PropertyMetadata(null));
```

Comment:
What's the purpose of storing these values? Performance? Easier code? Would it be better to store just the mask and check for slash each time?

Reply:
@skendrot you can compute both `EscapedMask` and `EscapedCharacterIndices` every time the text changes, I thought it would be better for performance to just store them somewhere. However, I do want your opinion on that because I am not sure storing these for every masked textbox is a good idea, what do you think?